devtools-feedback/devtools.js

```diff
@@ -0,0 +1,2 @@
+// Create a tab in the devtools area
+chrome.devtools.panels.create("GitHub Markdown a11y", "./devtools-feedback/toast.png", "./devtools-feedback/panel.html"); 
```

Comment:
Should we use "Mona's Markdown a11y" everywhere instead of "GitHub Markdown a11y"?